### PR TITLE
add flex-layout-attribute@1.0.3 w/ npm auto-update

### LINKED ISSUE #8035: [Request] Add flex-layout-attribute
**Library name:** flex-layout-attribute
**Git repository url:** https://github.com/StefanKovac/flex-layout-attribute/

### PR DESCRIPTION
**Pull request by nolsherry**
@pvnr0082t please help me to review this pull request, thanks!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/StefanKovac/flex-layout-attribute.git
 * Official website (optional, not the repository):http://progressivered.com/fla
 * NPM package url (optional): https://www.npmjs.com/package/flex-layout-attribute
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 375
   - Count of watchers: 22
   - Count of forks: 29
 * NPM download stats (optional):
   - Downloads in the last day: 7
   - Downloads in the last week: 156
   - Downloads in the last month: 205

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [X] No already exist / duplicated issue and PR
 * [X] The lib has notable popularity
   * [X] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [X] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [X] Has valid tags for each versions (for git auto-update)
 * [X] Auto-update setup
 * [X] Auto-update target/source is valid.
 * [X] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [X] The parent of the commit(s) in the PR is not old than 3 days.
 * [X] Pull request is sending from a non-master branch with meaningful name.
 * [X] Separate unrelated changes into different commits.
 * [X] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [X] Close corresponding issue in commit message
 * [X] Mention related issue(s), people in commit message, comment.

close #8035, cc @StefanKovac

### CODE REVIEW

File: ajax/libs/flex-layout-attribute/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "flexbox",
+    "layout",
+    "attribute",
+    "css",
+    "fla"
+  ],
+  "npmName": "flex-layout-attribute",
+  "npmFileMap": [
+    {
+      "basePath": "css",
```
Comment (pvnr0082t) [inline ajax/libs/flex-layout-attribute/package.json]: @nolsherry It's better not to use `css` or `js` as bathPath.

**Comment by pvnr0082t** [comment]
@nolsherry The latest version is v1.0.2 but the version info in files show v1.0.1. I think author just put the wrong file on npm. But there is only version not the same between v1.0.2 and v1.0.1. Maybe you can grab files with correct version info from GitHub.

File: ajax/libs/flex-layout-attribute/package.json (hunk 1 above)
Comment (nolsherry) [inline ajax/libs/flex-layout-attribute/package.json]: @pvnr0082t  Can I use this form ? thanks?
``` "files": [
      "**/flex-layout-attribute*"
  ]```

Comment (pvnr0082t) [inline ajax/libs/flex-layout-attribute/package.json]: Nope. I think that is not a good one. Please try others.

**Comment by nolsherry** [comment]
@pvnr0082t I have download the correct version from github, please help me to review this pull request, thanks!

**Comment by pvnr0082t** [comment]
@nolsherry What's the reason to use git auto-update?

**Comment by pvnr0082t** [comment]
@nolsherry There are duplicate files.

**Comment by nolsherry** [comment]
@pvnr0082t Because the version of files which downloaded from npm were wrong, I choose to download the files from git.

**Comment by PeterDaveHello** [comment]
@nolsherry What do you mean wrong here.

**Comment by nolsherry** [comment]
@PeterDaveHello I mean that the latest version is v1.0.2 but the version info in ```flex-layout-attribute.min.css```and```flex-layout-attribute.css``` which downloaded from npm show v1.0.1.

**Comment by PeterDaveHello** [comment]
@nolsherry Can you open an issue to confirm that?

**Comment by nolsherry** [comment]
@PeterDaveHello No problem, I will open an issue to confirm the problem.

**Comment by PeterDaveHello** [comment]
@nolsherry Thanks.

**Comment by nolsherry** [comment]
@pvnr0082t I have remove duplicate files, please help me to review this pull request, thanks!

**Comment by nolsherry** [comment]
@pvnr0082t I have asked the author about the latest version problem and I have updated to the latest version  1.0.3, please help me to review this pull request, thanks!

File: ajax/libs/flex-layout-attribute/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "css",
+    "fla"
+  ],
+  "npmName": "flex-layout-attribute",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "css/flex-layout-attribute*",
+        "css/maps/*"
```
Comment (pvnr0082t) [inline ajax/libs/flex-layout-attribute/package.json]: @nolsherry I think we can use `css/**/*.+(css|map)`

**Comment by nolsherry** [comment]
@pvnr0082t I have modified auto-update config, please help me to review this pull request, thanks!

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

**Comment by PeterDaveHello** [comment]
Nope, don't use `**/*` in non-dist folder, try `"css/**/*flex-layout-attribute*"`

**Comment by nolsherry** [comment]
@pvnr0082t I have modified ```css/**/* ```  to```css/**/*flex-layout-attribute*``` in ```files``` filed, please help me review this pull request, thanks!

**Comment by pvnr0082t** [comment]
@nolsherry Please rebase to the latest master branch.

**Comment by nolsherry** [comment]
@pvnr0082t I have rebased it to the latest master branch, please help me review this pull request, thanks!

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

**Pull request merged**